src/styles/styles.css

```diff
@@ -0,0 +1,306 @@
+  font-size: 0.813rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+}
+
+.new-book-wrapper {
+  width: 100%;
+  margin: 0;
+  align-items: baseline;
+}
+
+.book-wrapper {
+  display: flex;
+  flex-direction: row;
+  gap: 2.5rem;
+}
+
+.book-category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin-left: 2rem;
+  opacity: 0.5;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.book-title {
+  width: 30rem;
+  height: 1.813rem;
+  margin-left: 2rem;
+  font-family: RobotoSlab Arial, Helvetica, sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin-left: 2rem;
+  margin-top: 0.1rem;
+  font-family: RobotoSlab Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.book-buttons {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  border-left: solid 1px #e8e8e8;
+  margin-left: 2rem;
+  width: fit-content;
+}
+
+.books-btn {
+  width: 4.563rem;
+  height: 1.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: RobotoSlab Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  align-self: baseline;
+}
+
+.book-detail {
+  width: 45%;
+}
+
+.Oval-2 {
+  width: 4.25rem;
+  height: 4.25rem;
+  border-style: solid;
+  border-width: 5px;
+  border-image-source: linear-gradient(to bottom, var(--medium-blue), #379cf6);
+  border-image-slice: 1;
+}
+
+.circle {
+  width: 3rem;
+  height: 3rem;
+  background-color: transparent;
+  border-top: 0.3rem solid rgb(66, 139, 202);
+  border-right: 0.3rem solid rgb(66, 139, 202);
+  border-bottom: 0.3rem solid rgb(66, 139, 202);
+  border-left: 0.3rem solid rgba(66, 139, 202, 0.2);
+  border-radius: 30px;
+  transform: rotate(25deg);
+  margin-right: 0.625rem;
+}
+
+.percentage {
+  width: 25%;
+  align-items: center;
+  font-size: 1.375rem;
+  color: #121212;
+  height: 8.5rem;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.percentage .complete {
+  opacity: 0.5;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  font-size: 0.875rem;
+}
+
+.percentage .percent-num {
+  font-family: Montserrat Arial, Helvetica, sans-serif;
+  font-size: 2rem;
+}
+
+.Line-2 {
+  width: 0.0125rem;
+  height: 4.375rem;
+  margin: 1.125rem 3.688rem 0 4.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.chap-container {
+  width: 30%;
+}
+
+.current-chapter {
+  width: 10rem;
+  height: 1.125rem;
+  opacity: 0.6;
+  font-family: RobotoSlab Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  color: #121212;
+}
+
+.chapter-num {
+  width: 4.688rem;
+  height: 1.313rem;
+  font-family: RobotoSlab Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
```

Comment:
- [ ] Kindly fix the spacing/padding/margin issue between `Current Chapter`, `Chapter 17` and the `Button Update Progress` to match the given design.

![image](https://github.com/demesameneshoa/book-store/assets/60083437/2d5da096-a0fe-4247-b22c-ab8182c18083)